horace_core/symop/@SymopReflection/SymopReflection.m

```diff
@@ -0,0 +1,127 @@
+classdef SymopReflection < Symop
+
+    properties(Dependent)
+        u;
+        v;
+        normvec;
+    end
+
+    properties(Access=private)
+        u_;
+        v_;
+    end
+
+    methods
+        function obj = SymopReflection(u, v, offset)
+            if nargin == 0
+                return
+            end
+
+            if ~exist('offset', 'var')
+                offset = obj.offset;
+            end
+
+            if ~SymopReflection.check_args({u, v, offset})
+                error('HORACE:symop:invalid_argument', ...
+                      ['Constructor arguments should be:\n', ...
+                       '- Reflection: symop(3vector, 3vector, [3vector])\n', ...
+                       'Received: %s'], disp2str({u, v, offset}));
+            end
+
+            if norm(cross(u, v)) < 1e-2
+                error('HORACE:symop:invalid_argument', ...
+                      'Colinear vectors u & v')
+            end
+
+            obj.u = u;
+            obj.v = v;
+            obj.offset = offset;
+        end
+
+        function obj = set.u(obj, val)
+            if  ~obj.is_3vector(val)
+                error('HORACE:symop:invalid_argument', ...
+                      'Reflection vector u must be a three vector');
+            end
+            obj.u_ = val(:);    % make col vector
+        end
+
+        function u = get.u(obj)
+            u = obj.u_;
+        end
+
+        function obj = set.v(obj, val)
+            if  ~obj.is_3vector(val)
+                error('HORACE:symop:invalid_argument', ...
+                      'Reflection vector v must be a three vector');
+            end
+            obj.v_ = val(:);    % make col vector
+        end
+
+        function v = get.v(obj)
+            v = obj.v_;
+        end
+
+        function normvec = get.normvec(obj)
+            normvec = cross(obj.u, obj.v);
+            normvec = normvec / norm(normvec);
+        end
+
+        function selected = in_irreducible(obj, coords)
+        % Compute whether the coordinates in `coords` are in the irreducible
+        % set following the operation
+            selected = coords'*obj.normvec > 0;
+        end
+
+        function R = calculate_transform(obj, Minv)
+        % Get transformation matrix for the symmetry operator in an orthonormal frame
+        %
+        % The transformation matrix converts the components of a vector which is
+        % related by the symmetry operation into the equivalent vector. The
```

Comment:
the word equivalent gets used a lot in these descriptions, not sure it's right. I think you mean the transformed vector